SwiftUiMovieDb/Views/MoviePosterCard.swift

```diff
@@ -11,31 +11,33 @@ import Kingfisher
 struct MoviePosterCard: View {
     
     let movie: Movie
-    let cache = ImageCache.default
+    private let imageCornerRadius: CGFloat = 8
+    private let imageShadowRadius: CGFloat = 4
+    private let frameWidth: CGFloat = 204
+    private let frameHeight: CGFloat = 306
+    
     var body: some View {
         ZStack {
-            if cache.isCached(forKey: "my_cache_key") {
-              
-                Rectangle()
-                    .fill(Color.gray.opacity(0.3))
-                    .cornerRadius(8)
-                    .shadow(radius: 4)
-                
-                Text(movie.title)
-                .multilineTextAlignment(.center)
-            } else {
                 KFImage(movie.posterURL)
+                    .placeholder(posterPlaceholderImage)
                     .cacheMemoryOnly()
                     .resizable()
                     .aspectRatio(contentMode: /*@START_MENU_TOKEN@*/.fill/*@END_MENU_TOKEN@*/)
-                    .cornerRadius(8)
-                    .shadow(radius: 4)
-            }
+                    .cornerRadius(imageCornerRadius)
+                    .shadow(radius: imageShadowRadius)            
         }
-        .frame(width: 204, height: 306)
+        .frame(width: frameWidth, height: frameHeight)
     }
 }
 
+@ViewBuilder
```

Comment:
do we really need @ViewBuilder annotation here ? think about it and let me know why not ?